app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(team_list: list[Player]) -> int:
+    return sum([player.get_rating() for player in team_list])
```

Comment:
To improve code efficiency, avoid creating a redundant list by using a generator expression instead of a list comprehension inside the `sum` function. Change `[player.get_rating() for player in team_list]` to `(player.get_rating() for player in team_list)` as per the checklist recommendation.